devs2025.yml

```
@@ -28,19 +28,172 @@ translations:
+
+  - key: technology.workplace.yearly_income.takeaway.devs2025
+    t: |
+     США продовжують пропонувати набагато вищі зарплати порівняно з рештою світу.
+
+  - key: technology.workplace.income_evolution.takeaway.devs2025
+    t: |
+      Попри побоювання спаду галузі, лише 9% респондентів повідомили про зниження свого доходу протягом минулого року.
+
+  - key: technology.workplace.job_finding_application_count.takeaway.devs2025
+    t: |
+      Хоча респондентам загалом знадобилося в середньому 4 заявки на роботу, щоб знайти свою поточну роботу, це число зменшувалося зі збільшенням років досвіду.
+
+      Тим не менш, невелике збільшення спостерігалося на вищому кінці кривої досвіду, що можна пояснити віковою дискримінацією або застарілими навичками, які ускладнюють пошук роботи.
+
+  - key: technology.workplace.job_finding_methods.takeaway.devs2025
+    t: |
+      Наявність більш особистого зв'язку з компанією під час подання заявки (на відміну від використання дошки оголошень про роботу чи соціальних мереж) пов'язана з вищим доходом.
+
+      Єдиний виняток — це наймання через роботу з відкритим кодом, яка пов'язана з тими самими рівнями доходу, що й більш прямі методи, такі як рекрутери чи рекомендації.
+
+  - key: technology.workplace.remote_work_policy.takeaway.devs2025
+    t: |
+      Нестача віддаленої роботи корелює як із нижчим медіанним доходом, так і з нижчим вдоволенням роботою.
+
+  - key: technology.workplace.work_hours_per_week.takeaway.devs2025
+    t: |
+      Чим більше ви працюєте, тим більше заробляєте, принаймні до певної межі. Індія є найпрацьовитішою країною з медіанними 45 годинами роботи на тиждень, тоді як більшість інших країн зосереджені навколо позначки 40 годин.
+
+  - key: technology.workplace.job_happiness.takeaway.devs2025
+    t: |
+      Розробники загалом досить задоволені своєю роботою.
+
+  - key: technology.workplace.workplace_perks.takeaway.devs2025
+    t: |
+      Всупереч стереотипу асоціального комп'ютерного ботаніка, виявляється, що найбільше ми цінуємо насправді один одного — стосунки з колегами займають перше місце серед того, що розробники найбільше цінують у своєму робочому місці.
+
+  - key: technology.workplace.workplace_difficulties.takeaway.devs2025
+    t: |
+      Зрештою, всім нам потрібно заробляти на життя, і погана компенсація посіла перше місце як головна проблема робочого місця, з якою розробники стикаються сьогодні.
+
+  # technology
+
+  # - key: technology.mobile_os.takeaway.devs2025
+  #   t: |
+  #     xxx
+
+  # - key: technology.desktop_os.takeaway.devs2025
+  #   t: |
+  #     xxx
+
+  # - key: technology.programming_languages.takeaway.devs2025
+  #   t: |
+  #     xxx
+
+  - key: technology.productivity_apps.takeaway.devs2025
+    t: |
+      Коли йдеться про додатки для продуктивності, варто відзначити вражаючий результат **Obsidian**, який посів 6-те місце загалом, попри те, що навіть не був частиною попередньо визначених варіантів питання.
```

```suggestion
      Коли йдеться про застосунки для продуктивності, варто відзначити вражаючий результат **Obsidian**, який посів 6-те місце загалом, попри те, що навіть не був частиною попередньо визначених варіантів питання.
```
додатки - калька (у значенні застосунки)